system-int-test/test/cmr/system_int_test/ingest/subscription_ingest_test.clj

```diff
@@ -22,12 +22,12 @@
                                                                   [:read :update])]))
 
 (deftest subscription-ingest-on-prov2-test
```

Comment:
We are missing tests on specific ACL permission for update/delete. e.g. 1) with Ingest Management ACL update permission, but no Subscription Management ACL update permission trying to update a subscription concept. 2) with  Subscription Management ACL update permission, but no  Ingest Management ACL update permission trying to update a subscription concept, and.delete a subscription concept etc. Will these tests be added in a later PR?

Reply:
Ingest Management ACL permissions were tested before when the subscription was first added. Now we want to add the Subscription Management ACL on top of Ingest Management ACL, so the tests have Ingest Management permissions granted already, just test the Subscription ACL permissions. I don't think it's necessary to test all the permutations, You test one, when it's done, you can just grant that permission, then test the other.

Reply:
provider_ingest_permissions_test.clj contains the tests for Ingest Management ACL.  For Subscription Management ACL, it's covered by subscription_ingest_tests.  Whenever you add something new, you test on top of what's already there, rather than testing all the permutations of all the existing ACLs, validations.

Reply:
Done.